templates/secret-db-access.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}-db-access
 type: Opaque
 data:
-  {{- $secretObj := (lookup "v1" "Secret" .Release.Namespace .Values.db_access.secret_name) | default dict }}
+  {{- $secretObj := (lookup "v1" "Secret" .Release.Namespace .Values.db_access.unified_secret_name) | default dict }}
   {{- $secretData := (get $secretObj "data") | default dict }}
   {{- $secretHost := (get $secretData "host") | default (printf "%s-%s" (include "zabbix.fullname" .) "postgresql" | b64enc) }}
   {{- $secretPort := (get $secretData "port") | default (.Values.postgresql.service.port | toString | b64enc) }}
```

Comment:
Hi @sa-ChristianAnton 

The parameter `postgresql.service.port` will it continue to exist?

Reply:
Well, yes, at least I have implemented like this. Do you think we should eliminate this var and default to just 5432?

Reply:
I thought we would no longer refer to the parameter ``postgresql.service.port`` but to the ``.Values.db_access.unified_secret_name``. It makes sense?

Reply:
I still don't get it, sorry.

Inside the Secret, when it gets generated from scratch (`db_access.unified_secret_autocreate`), I would set the port to "5432" or whatever else is set in the `postgresql.service.port` setting. Secret autocreation does only make sense if `postgresql.enabled=true`, as with an external database you would anyway have to create a secret taking the data of the existing (external?) database or already have such (pgo operator), so from my point of view it is absolutely fine to refer to `postgresql.service.port` as a default here.

Am I missing anything?